src/platforms/javascript/common/configuration/integrations/plugin.mdx

```diff
@@ -53,6 +52,16 @@ Available options:
 
 <PlatformContent includePath="configuration/debug" />
 
+### Offline
+
+_Import name: `Sentry.Integrations.Offline`_
+
+This integration uses the web browser's [online and offline events](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/Online_and_offline_events) to detect when no network connectivity is available. If offline, it saves events to the web browser's client-side storage (typically IndexedDB) then automatically uploads events when network connectivity is restored.
+
+This plugin does not attempt to provide local storage or retries for other scenarios; for example, if the browser has a local area connection but no Internet connection, then it may report that it's online, and Sentry's `Offline` plugin will not attempt to save or retry any send failures in this case.
```

Comment:
Suggest the following to adhere to style guide for certain words and to avoid semi-colons
```suggestion
This plugin does not attempt to provide local storage or retries for other scenarios. For example, if the browser has a local area connection but no internet connection, then it may report that it's online, and Sentry's `Offline` plugin will not attempt to save or retry any send failures in this case.
```